e2e/cypress/integration/messaging/message_channel_draw_spec.js

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import * as TIMEOUTS from '../../fixtures/timeouts';
+
+describe('Draw plugin : Post message', () => {
+    before(() => {
+        //Login with admin access -> enable Draw plugin
+        cy.apiLogin('sysadmin').visit('/');
+        cy.switchToPluginManagementEnableDisableDraw('Enable', fileName, fileType);
```

Comment:
Since the function "switchToPluginManagementEnableDisableDraw" is not used in other tests, it's better to include this function within this test itself. This way we do not overload the ui_commands with functions used only once.

Reply:
Functions specific to test are added at spec level